client/components/contributepage/HeroSection.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styled from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+const Wrapper = styled.div`
```

Comment:
The padding on this section seems a bit big. I know the Hero on the main page is a large section, but I think we are going towards smaller sections on sub-pages. Something like `padding: 20px 0;`

Reply:
:100: